demos-europe/demosplan-core
# chore(refs dplan-12315): Replace draggable with sortable in newsAdminList

### Ticket
DPLAN-12315




### How to review/test


Login as FPA
goto /verfahren/<procedure-id>verwalten/aktuelles

Drag drop articles and see if everything work as expected

Requires: f_wip_vue3_draggables in demosplan-ui





### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
